src/App.tsx

```diff
@@ -1,9 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export function filterMovies(query: string) {
+  const formattedQuery = query.toLowerCase();
+  const includesQuery = (source: string) => (
+    source.includes(formattedQuery)
+  );
+
+  return (moviesFromServer.filter((movie) => {
+    const { title, description } = movie;
+    const formattedTitle = title.toLowerCase();
+    const formattedDescription = description.toLowerCase();
+
+    return (includesQuery(formattedTitle)
+    || includesQuery(formattedDescription));
+  }));
```

Comment:
Instead of doing it here use .toLowerCase in the `includesQuery` function, it will be easier to manager later because you don't need to remember and manually lower case it every time, the function should take care of it

```suggestion
  return (moviesFromServer.filter((movie) => {
    const { title, description } = movie;

    return (includesQuery(title)
    || includesQuery(description));
  }));
```

Reply:
It lowercases it so that the search is case insensitive, as per the instruction in readme. By removing lowercasing, it causes the following:
![image](https://github.com/mate-academy/react_movies-list-filter/assets/55781950/532c4b8d-86a6-41e0-81d8-bd4a7f27aa1d)